### ISSUE TITLE
Add authentication to API requests

### ISSUE THREAD

**Issue by zhalloran** [addisclinic/mobile-dispatch-server#8]
We should, at some point, add authentication tests for all API requests.

**Comment by tribbettz** [comment]
Agreed. I talked to Albert about this and ultimately it comes down to priorities, etc. So once codebases are synced and notification endpoint is functional with a little refactoring thrown in, we should discuss the next set of tasks which could include (proper) authentication. I created another issue for tokenization & sessions (which OpenMRS supports) so that could be a good next step. But I'll leave that to you and Lydia and our weekly calls in deciding :)
